docs/mobility/slam/cartographer/_index.md

```diff
@@ -78,36 +78,47 @@ See Viam's [Pricing](https://www.viam.com/product/pricing) for more information.
 To create a new map, follow the instructions below.
 Creating a new map uses an instance of the cartographer module running in the cloud.
 
-1. Configure your `cartographer` SLAM service
+1. Enable data capture and configure your `cartographer` SLAM service
 
-   After installing your physical RPlidar and adding the `rplidar` module as outlined in the [requirements](#requirements) section, follow the steps below to add the `cartographer` module to your machine:
+   After installing your physical RPlidar and adding the `rplidar` module as outlined in the [requirements](#requirements) section, follow the steps below to enable data capture and add the `cartographer` module to your machine:
 
    {{< tabs name="Create new map">}}
    {{% tab name="Config Builder" %}}
-   Follow the instructions below to set up the `cartographer` module on your machine:
 
-   1. Navigate to the **Config** tab of your machine's page in [the Viam app](https://app.viam.com).
-   1. Click on the **Services** subtab and click **Create service** in the lower-left corner.
-   1. Select **SLAM**, then select `cartographer`.
+   Add the data management service:
+
+   1. From your machine's **Config** tab, navigate to the **Services** subtab.
+   2. Click **Create service** in the lower-left corner of the page.
+      Choose `Data Management` as the type and specify a name for your data management service, for example `Data-Management-Service`.
+   3. Click **Create**.
+   4. On the panel that appears, you can manage the capturing and syncing functions and specify the **directory**, the sync **interval** and any **tags** to apply to captured data. See the [data management service](/data/) for more information.
+
+   Enable data capture for your camera and/or movement sensor:
+
+   5. Find the component's card on your machine's **Config** tab.
+   6. Click `Add Method` and then select the method type and the capture frequency. For an RPlidar, choose the method [`NextPointCloud`](/components/camera/#getpointcloud).
+      We recommend a capture frequency of `5` Hz for RPlidar cameras and `20` Hz for movement sensors.
+
```

Comment:
What do you think of moving the alert that says:

```
Be aware that data is not only captured when a slam session is running.
[Data Capture](/data/capture/) continuously monitors and captures your machine's sensor data while the machine is running.
To avoid incurring charges while not in use, [turn off data capture for your sensors](/data/capture/).
```

up to here, next to where we now talk about adding data capture?

Reply:
Sounds good, moved